guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -200,7 +200,7 @@ If you use a {SmartProxyServer} FQDN for the `proxy.url` argument, ensure that y
 
 [options="nowrap" subs="+quotes"]
 ----
-proxy.url=https://_capsule.example.com_:9090 proxy.type=proxy
+proxy.url=https://_{smartproxy.example.com}_:9090 proxy.type=proxy
```

Comment:
Noted. I think we need to create a `katello` env too. @lzap   @melcorr what do you think? like `satellite` and `foreman` we can have another section of variables in the `katello` env.

Reply:
So, @ekohl in the above example, we would have to potentially display two separate port numbers, depending on whether the deployment was with or without Katello?

Reply:
Yes. Satellite is basically Katello + branding + a few other plugins.

Reply:
Interesting @ekohl. So we need attributes for differences between Satellite and Foreman, and then Foreman with and without Katello. The more of these we can identify at an early stage, the better. Thank you very much @ekohl

Reply:
This is an issue because this is upstream docs for both Katello and Foreman. What I suggest:

* introduce new var: `{smartproxy_port}` with 8443/9090
* under every use we need to put remark: When installed in Katello scenario, use port 9090.

Reply:
@lzap ++

Reply:
Notes from the meeting:

* use `{proxy_https_port}` variable
* check if we need the same for HTTP (lzap to find out)
* put note everywhere: "For katello scenario deployment, use port 9090" and show it only if upstream

Reply:
So HTTP port is 8000 for both upstream and downstream. No need for a variable.